File: Src/Icm.ContextConsole/Context/IActionFinder.vb
Imports System.Linq
Imports Icm.Reflection
Imports Icm.Localization
Imports Icm.Ninject

''' <summary>
''' An Action Finder finds the actions a context implements
''' </summary>
''' <remarks></remarks>
Public Interface IActionFinder
    Function GetActions(ctl As IContext) As IEnumerable(Of IAction)
End Interface

File: Src/Icm.ContextConsole/Context/IActionFinder.cs
using System.Collections.Generic;

/// <summary>
/// An Action Finder finds the actions a context implements
/// </summary>
/// <remarks></remarks>
public interface IActionFinder
{
	IEnumerable<IAction> GetActions(IContext ctl);
}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================
